aiida/orm/nodes/node.py

```diff
@@ -175,13 +175,15 @@ def initialize(self) -> None:
         self._incoming_cache = list()
 
     def _validate(self) -> bool:
-        """Check if the attributes and files retrieved from the database are valid.
+        """Validate information stored in Node object.
 
-        Must be able to work even before storing: therefore, use the `get_attr` and similar methods that automatically
-        read either from the DB or from the internal attribute cache.
+        For the :py:class:`~aiida.orm.Node` base class, this check is always valid.
+        Subclasses can override this method to perform additional checks
+        and should usually call ``super()._validate()``!
```

Comment:
Maybe this is obvious if you are used how the `super` works, but I'd still keep this clarification:

```suggestion
        and should usually call ``super()._validate()`` first!
```

or

```suggestion
        and should usually call ``super()._validate()`` at the beginning of it!
```